docs/content/Getting-Started/Cloud/05-Add-a-pre-aggregation.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Add a pre-aggregation
+permalink: /getting-started/cloud/add-a-pre-aggregation
+category: Getting Started
+subCategory: Cube Cloud
+menuOrder: 6
+isDisableFeedbackBlock: true
+---
+
+From the Developer Playground, execute a query:
+
+<div style="text-align: center">
+  <img
+    alt="Cube Cloud Deployment Developer Playground Screen showing query results"
+    src="https://ucarecdn.com/d2210bb7-a35e-4da7-ba29-447da24cc06d/"
+    style="border: none"
+    width="80%"
+  />
+</div>
+
+Just above the results, click on **Query was not accelerated with
+pre-aggregation** to bring up the Rollup Designer:
+
+<div style="text-align: center">
+  <img
+    alt="Cube Cloud Deployment Developer Playground Screen showing Rollup Designer"
+    src="https://ucarecdn.com/76c685b5-f778-486c-bf3f-e06996e29a50/"
+    style="border: none"
+    width="80%"
+  />
+</div>
+
+The Rollup Designer will automatically suggest a pre-aggregation for the query;
+simply click **Add to the Data Schema** and then retry the query in the
+Playground. This time, the query should be accelerated with a pre-aggregation:
```

Comment:
Let's put an emphasis on the query latency. It's now two-digit in ms, much better.

Reply:
Can we discuss this in a Slack huddle?